### ISSUE TITLE
How do I disable log?

### ISSUE THREAD

**Issue by jhemmmm** [pascalbaljetmedia/laravel-ffmpeg#173]
I've been receiving quite a lot of logs.

For example:

ffmpeg executed command successfully
--

**Comment by KoVaLsKy2010** [comment]
up

**Comment by pascalbaljet** [comment]
I've rewritten the package and it has a configuration setting to disable the logging. Could you test the `v7` branch? Almost no breaking changes except for some filter stuff in HLS exports.

https://github.com/pascalbaljetmedia/laravel-ffmpeg/tree/v7

`composer require "pbmedia/laravel-ffmpeg:v7.x-dev"`

In the `laravel-ffmpeg.php` config file set `enable_logging` to `false`:

```php
<?php

return [
    'ffmpeg' => [
        'binaries' => env('FFMPEG_BINARIES', 'ffmpeg'),
        'threads'  => 12,
    ],

    'ffprobe' => [
        'binaries' => env('FFPROBE_BINARIES', 'ffprobe'),
    ],

    'timeout' => 3600,

    'enable_logging' => false,
];

```

**Comment by pascalbaljet** [comment]
Fixed in `v7.0`
